h2o-py/h2o/explanation/_explain.py

```diff
@@ -2271,7 +2271,7 @@ def explain(
     or a variable importance plot.  Most of the explanations are visual (plots).
     These plots can also be created by individual utility functions/methods as well.
 
-    :param models: H2OAutoML object, H2OModel, or list of H2O models
+    :param models: H2OAutoML object, supervised H2O model, or list of supervised H2O models
```

Comment:
unrelated - would it work on a imported mojo? "GenericModel"?

Reply:
It does but it wasn't intended to. It doesn't plot the SHAP_* related plots as it assumes only tree-based models are able to calculate SHAP and the other plots seem to work correctly. And AFAIK MOJOs don't support `predict_contributions` (at least my quick attempt to do so failed) so "unofficially" I'd say explain supports MOJOs as much as it can but we don't run tests on MOJOs and it depends on the individual parts, e.g., if there is some limitation for MOJOs on PDP calculation it will be here as well etc.